### PR TITLE
Fix comment for Var

### PR DESCRIPTION
**Pull request by biboudis**

### CODE REVIEW

File: library/src-bootstrapped/scala/quoted/util/Var.scala [hunk 1]
```diff
@@ -17,21 +17,21 @@ object Var {
   /** Create a variable initialized with `init` and used in `body`.
    *  `body` recieves a `Var[T]` argument which exposes `get` and `update`.
    *
-   *  `var`('(7)) {
+   *  Var('(7)) {
    *    x => '{
-   *      while(0 < ~x)
-   *        ~x.update('(~x - 1))
-   *      ~x.get
+   *      while(0 < $x.get)
```
Comment (nicolasstucki) [inline library/src-bootstrapped/scala/quoted/util/Var.scala]:
```suggestion
   *      while(0 < ${x.get})
```

File: library/src-bootstrapped/scala/quoted/util/Var.scala [hunk 2]
```diff
@@ -17,21 +17,21 @@ object Var {
   /** Create a variable initialized with `init` and used in `body`.
    *  `body` recieves a `Var[T]` argument which exposes `get` and `update`.
    *
-   *  `var`('(7)) {
+   *  Var('(7)) {
```
Comment (nicolasstucki) [inline library/src-bootstrapped/scala/quoted/util/Var.scala]:
```suggestion
   *  Var('{7}) {
```

File: library/src-bootstrapped/scala/quoted/util/Var.scala [hunk 3]
```diff
@@ -17,21 +17,21 @@ object Var {
...
    *    x => '{
-   *      while(0 < ~x)
-   *        ~x.update('(~x - 1))
-   *      ~x.get
+   *      while(0 < $x.get)
+   *        $x.update('($x.get - 1))
```
Comment (nicolasstucki) [inline library/src-bootstrapped/scala/quoted/util/Var.scala]:
```suggestion
   *        ${x.update('{${x.get} - 1})}
```

File: library/src-bootstrapped/scala/quoted/util/Var.scala [hunk 4]
```diff
@@ -17,21 +17,21 @@ object Var {
...
+   *      $x.get
```
Comment (nicolasstucki) [inline library/src-bootstrapped/scala/quoted/util/Var.scala]:
```suggestion
   *      ${x.get}
```

File: library/src-bootstrapped/scala/quoted/util/Var.scala [hunk 5]
```diff
@@ -17,21 +17,21 @@ object Var {
...
    *    }
    *  }
    *
-   *  will create the equivalent of
+   *  will create the equivalent of:
    *
-   *  '{
+   *  {
```
Comment (nicolasstucki) [inline library/src-bootstrapped/scala/quoted/util/Var.scala]:
```suggestion
   *  '{
```

Comment (biboudis) [reply library/src-bootstrapped/scala/quoted/util/Var.scala]: 😆

Comment (nicolasstucki) [reply library/src-bootstrapped/scala/quoted/util/Var.scala]: Just because it returns an `Expr[_]`

**Pull request merged**